src/errors/pubnub-api-error.ts

```diff
@@ -58,7 +58,9 @@ export class PubNubAPIError extends Error {
       category = StatusCategory.PNNetworkIssuesCategory;
       message = 'Network issues';
     } else if (errorName === 'TypeError') {
-      category = StatusCategory.PNBadRequestCategory;
+      if (message.indexOf('Load failed') !== -1 || message.indexOf('Failed to fetch') != -1)
```

Comment:
👍🏻

Reply:
a dev to dev comment!
 What about using `includes()` instead of `indexOf`.

Reply:
@mohitpubnub `fetch` errors not really talkative and don't provide any insights on what exact error it was (and looks like it is deliberately). So I've found errors which can be generated by `fetch` on macOS (`Load failed`) and win (apparently it is `Failed to fetch`)

Reply:
Yeah!! It's looks a bit challenging to match more accurate error message clauses.!